examples/templates/aws-ecs-container/main.tf

```diff
@@ -117,6 +119,40 @@ resource "coder_agent" "coder" {
     curl -fsSL https://code-server.dev/install.sh | sh -s -- --method=standalone --prefix=/tmp/code-server --version 4.8.3
     /tmp/code-server/bin/code-server --auth none --port 13337 >/tmp/code-server.log 2>&1 &
   EOT
+
+  metadata {
+    key          = "cpu"
+    display_name = "CPU Usage"
+    interval     = 5
+    timeout      = 5
+    script       = <<-EOT
+      #!/bin/bash
+      set -e
+      top -bn1 | grep "Cpu(s)" | awk '{print $2 + $4 "%"}'
+    EOT
+  }
+  metadata {
+    key          = "memory"
+    display_name = "Memory Usage"
+    interval     = 5
+    timeout      = 5
+    script       = <<-EOT
+      #!/bin/bash
+      set -e
+      free -m | awk 'NR==2{printf "%.2f%%\t", $3*100/$2 }'
+    EOT
+  }
```

Comment:
Memory and CPU usage for container-based templates (e.g. Docker, Kubernetes, ECS, etc) will be inaccurate since it'll show the entire host, not the group. Not sure what Fly shoes, but I'd be surprised if that is accurate too.

https://fabiokung.com/2014/03/13/memory-inside-linux-containers/

Reply:
I believe @ammario is working on a solution in #6938

Reply:
Fly should be accurate since it's a true VM.

And, we may want to avoid putting in this metadata on the containers since it's misleading.

Reply:
Sure. I will remove stats from docker/containers templates.